msgcntr/messageforums-app/src/webapp/js/sak-10625.js

```diff
@@ -2,8 +2,12 @@
 var click=0;
 sak10625_disabler = function(){
   var existing_event = this.onclick;
+  $("a, input[type=button], input[type=submit]").attr("onclick", "");
```

Comment:
I believe jQuery.off is the preferred method in jQuery 1.6+

http://api.jquery.com/off/